### PR TITLE
Fix typos `brackets`->`bracket` in manual

### PR DESCRIPTION
**Pull request by johnomotani**

### CODE REVIEW

**Review by ZedThree — APPROVED** [review]
Nice spot, thanks @johnomotani !

File: manual/sphinx/user_docs/differential_operators.rst [hunk 1]
```diff
@@ -430,7 +430,7 @@ is the background *equilibrium* magnetic field.
 |                  | derivatives. The `Laplacian` solver performs the   |
 |                  | inverse operation                                  |
 +------------------+----------------------------------------------------+
-| ``brackets``     | Poisson brackets. The Arakawa option, neglects the |
+| ``bracket``     | Poisson brackets. The Arakawa option, neglects the |
```
Comment (bendudson) [inline manual/sphinx/user_docs/differential_operators.rst:433]: Does it need another space to line up `|` bars? Not sure how picky RST is.

Comment (ZedThree) [reply manual/sphinx/user_docs/differential_operators.rst:433]: Also good spot! Readthedocs builds the PR now, so we'll be able to check before merging

Comment (johnomotani) [reply manual/sphinx/user_docs/differential_operators.rst:433]: > Does it need another space to line up | bars? Not sure how picky RST is.

Good spot, could do with that even if RST doesn't complain...
```suggestion
| ``bracket``      | Poisson brackets. The Arakawa option, neglects the |
```

Comment (johnomotani) [reply manual/sphinx/user_docs/differential_operators.rst]: ...and in the PR docs build, the table disappeared, so I guess the space was necessary.

**Comment by johnomotani** [comment]
That looks better. Thanks @bendudson!

Sorry I forgot to use `[skip ci]` again. Need to get used to that!

**Pull request merged**